### PR TITLE
[BOP-583] Known Issue insertion: Initial commit for configuration upgrade KI.

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**

### CODE REVIEW

File: content/docs/release-notes/known-issues.md [hunk 1]
```diff
@@ -39,4 +39,18 @@ command is ignored.
 
    ```sh
    mkectl apply
-   ```
\ No newline at end of file
+   ```
+
+## [BOP-583] LDAP settings fail to migrate during upgrade from MKE 3
```
Comment (nwneisen) [inline content/docs/release-notes/known-issues.md:44]: Users wont know jira numbers so I don't think we need to include it here

File: content/docs/release-notes/known-issues.md [hunk 2]
```diff
@@ -39,4 +39,18 @@ command is ignored.
...
+
+LDAP configurations are not stored in MKE 3 configuration files, and thus they
+are not included when you upgrade to MKE 4 from an MKE 3 installation.
+
+**Workaround:**
+
+When upgrading from MKE 3, you must manually add the LDAP configuration.
+
+1. Make a request to https://{{host}}/enzi/v0/config/auth/ldap on the MKE 3
```
Comment (nwneisen) [inline content/docs/release-notes/known-issues.md:53]: This should point to the mke3 docs that talk about working with the configs through the api

File: content/docs/release-notes/known-issues.md (hunk 1 above)
Comment (KoryKessel-Mirantis) [reply content/docs/release-notes/known-issues.md:44]: No, but it helps us to internally track the Known Issues, and we have already started down the path. We can resolve to remove them at a later date in a separate PR.

**Review by nwneisen — APPROVED** [review]

**Comment by nwneisen** [comment]
Rebase and then it LGTM

**Pull request merged**